Backend/Services/UserService.cs

```diff
@@ -15,10 +15,25 @@ public UserService(IUserRepository repository)
     {
         _repository = repository;
     }
+    
+    public IEnumerable<User> GetAllUsers()
+    {
+        return _repository.GetAll();
+    }
 
     public void CreateUser(User user) => _repository.Create(user);
 
-    public User? GetUserById(int id) => _repository.Get(id);
+    public User? GetUserById(int id)
+    {
+        var user = _repository.Get(id);
+
+        if (user == null)
+        {
+            return null;
+        }
+
+        return user;
+    }
```

Comment:
`public User? GetUserById(int id) => _repository.Get(id);` would do the same

Reply:
Fixed